common/content/commands.js

```diff
@@ -1096,10 +1096,12 @@ const Commands = Module("commands", {
                 }
                 else {
                     function completerToString(completer) {
-                        if (completer)
-                            return [k for ([k, v] in Iterator(completeOptionMap)) if (completer == completion[v])][0] || "custom";
-                        else
+                        if (completer) {
+                            return Object.keys(completeOptionMap)
+                                         .filter(k => (completer == completion[completeOptionMap[k]]))[0] || "custom";
```

Comment:
can the `custom` go on a separate line? Would make it easier to understand that it's an `||` to the filter expression.